Header header

string   interface_name
string   human_readable_ip_address # something like "192.168.1.33/24"
string   default_gateway
string   dns_addresses
bool     link_is_up
string   wlan_ssid
